README.md

```diff
@@ -11,6 +11,11 @@ Simply press a key (`<C-l>` in insert mode by default) and select what you want
 
 `insert_link` only works for elements in the current document.
 
+### Automatic File Link Insertion
+Simply press a key (`<C-f>` in insert mode by default) and select what Neorg file you want to link to.
+
+`insert_file_link` only works for elements in the current document.
```

Comment:
not sure what you mean by that
it doesn't link to elements in the current document
did you perhaps mean `workspace`?
```suggestion
`insert_file_link` only works for files in the current workspace.
```

Reply:
copy-patsa, removed

Reply:
wouldn't remove it but change it to what I suggested